src/api/treeTrackerApi.js

```diff
@@ -68,51 +71,75 @@ export default {
       handleError(error);
     }
   },
-  approveCaptureImage(id, morphology, age, captureApprovalTag, speciesId) {
+  approveCaptureImage(capture, morphology, speciesId, captureApprovalTag) {
     try {
-      const query = `${API_ROOT}/api/${getOrganization()}trees/${id}`;
+      log.debug(
+        'approveCaptureImage',
+        capture,
+        morphology,
+        speciesId,
+        captureApprovalTag
+      );
 
-      log.debug(query);
+      // map legacy data to fields for new microservice
+      const newCapture = {
+        id: capture.uuid,
+        reference_id: capture.id,
+        session_id: capture.session_id, // no legacy equivalent
+        grower_account_id: capture.planterId,
```

Comment:
You're correct, it won't be handled by the treetracker api, but I think it would be needed by the field-data-service to identify the correct grower and the grower_account_id could be added that way.  @Kpoke Your thoughts?

Reply:
The field-data service already has that information. The wallet_registrations table has that column populated.